src/main/kotlin/pt/up/fe/ni/website/backend/model/Event.kt

```diff
@@ -11,11 +11,19 @@ import pt.up.fe.ni.website.backend.model.constants.EventConstants as Constants
 @Entity
 class Event(
     @JsonProperty(required = true)
-    @field:Size(min = Constants.Title.minSize, max = Constants.Title.maxSize)
+    @field:Size(
+        min = Constants.Title.minSize,
+        max = Constants.Title.maxSize,
+        message = "size must be between ${Constants.Title.minSize} and ${Constants.Title.maxSize}"
+    )
```

Comment:
I don't think we need to override the `Size`'s message, since the default already works well for us. What do you think?

This also applies to the other models